php/utils-wp.php

```diff
@@ -249,11 +249,11 @@ function wp_get_cache_type() {
 			$message = 'APC';
 
 			// Test for Redis Object Cache (https://github.com/alleyinteractive/wp-redis)
-		} elseif ( isset( $wp_object_cache->redis ) && is_a( $wp_object_cache->redis, 'Redis' ) ) {
+		} elseif ( isset( $wp_object_cache->redis ) && ( $wp_object_cache->redis instanceof \Redis ) ) {
```

Comment:
Parentheses are not needed here:
```suggestion
		} elseif ( isset( $wp_object_cache->redis ) && $wp_object_cache->redis instanceof \Redis ) {
```